test/TestDidDocuments.ts

```diff
@@ -0,0 +1,139 @@
+import {IDIDDocument} from "@ew-did-registry/did-resolver-interface"
+
+
+export const firstDocument : IDIDDocument = {
+    id: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3',
+    service: [
+      {
+        id: '78ad468a-d964-4ca6-b496-c7fabd526682',
+        did: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3',
+        iat: 1621600642888,
+        iss: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3',
+        sub: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3',
+        hash: '1e8ac4cfc7a405924d146b395f03919d0fedd646371297f5e79dd7ec7397e54a',
+        signer: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3',
+        hashAlg: 'SHA256',
+        profile: [Object],
+        serviceEndpoint: 'QmT7d8ntKhgJb3hqCyXscYxDPdaM3asXKenYyJcxuxoxcF'
+      },
+      {
+        id: '7ef471ed-0a37-4ba5-98f0-75eb90d64314',
+        did: 'did:ethr:0x731ac21Aa72c1A6E15243AFBB34cA9CC073D419B',
+        iat: 1622662901815,
+        iss: 'did:ethr:0x829b91Fa3e91EA4448365ADA58C7Bad1Ff142866',
+        sub: 'did:ethr:0x731ac21Aa72c1A6E15243AFBB34cA9CC073D419B',
+        hash: '35405a628207f5f165d7c0ae6663d089733cea2a5233c4971d65699bbff2f45f',
+        fields: [],
+        signer: 'did:ethr:0x829b91Fa3e91EA4448365ADA58C7Bad1Ff142866',
+        hashAlg: 'SHA256',
+        claimType: 'prequalified.roles.flexmarket.apps.elia.iam.ewc',
+        serviceEndpoint: 'QmTUFHwSk7RbNUK8u2bwyDNtvmtxhk83JNCc1YnJvYxghd',
+        claimTypeVersion: '1.0.0'
+      }
+    ],
+    authentication: [
+      {
+        type: 'owner',
+        validity: [Object],
+        publicKey: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3#owner'
+      }
+    ],
+    created: null,
+    delegates: null,
+    proof: null,
+    publicKey: [
+      {
+        id: 'did:ethr:0x72EFf9faB7876c4c1b6cAe426c121358431758F3#key-owner',
+        type: 'Secp256k1veriKey',
+        controller: '0x72EFf9faB7876c4c1b6cAe426c121358431758F3',
+        publicKeyHex: '0x0355f6f95a109a1ebe2a2aaea74f49274ea55aaecd2d32a44a2002bcf63002b913'
```

Comment:
IMO, would be better instead of the string, if the variable was used so that it was clear to a reader of the code that the publicKey in this DID doc is from the private key that signed some message. (I might not have put this in the right spot, but hopefully it is clear 😄 )
```suggestion
        publicKeyHex: firstKeys.publicKey
```